### ISSUE TITLE
Cannot get Resources in API group

### ISSUE THREAD

**Issue by dnotari** [jcmoraisjr/haproxy-ingress#249]
Hey so questions, hoping someone can point me in the right direction.  I am new to Kubernetes in general my team and I are kind being thrown into it without much time to really gain the knowledge we need to this well.

So I used the the template in the instructions and changed the namespaces to match mine "dev".

Everything comes up but the HAProxy-Ingress pod goes into a crashloop.

running kubectl logs on the pod gives me the following error messages.

**no service with name dev/ingress-default-backend found: services "ingress-default-backend" is forbidden: User "system:serviceaccount:dev:ingress-controller" cannot get resource "services" in API group "" in the namespace "dev"**

So seems like a silly RBAC issue, but I haven't had any luck by changing or modify the permissions.  Anyone have any ideas or pointers?

**Comment by jcmoraisjr** [comment]
Hi, welcome to Kubernetes and to the HAProxy Ingress controller! I'd have a look at your `ingress-controller` cluster role binding, its `subjects[]/namespace` should be `dev`. See also the `ingress-controller` role binding - you should have one in the `dev` namespace where `subjects[]/namespace` should also be `dev`.

**Comment by dnotari** [comment]
Thank you @jcmoraisjr!  Wow was something so simple.  I not sure what happened, but when I did a kubectl -n dev get ClusterRoleBind, I saw the ingress-controller so I assumed it was good.  Doing a describe on it showed it was in the ingress-controller namespace.  Once updated my pod connected happily.  Not sure how that happened but I will remember in the future to check those details.

Thanks again for taking the time to answer my question.
